recipes/jkqtplotter/bld.bat

```diff
@@ -0,0 +1,9 @@
+:: Fix for https://github.com/jkriege2/JKQtPlotter/issues/35
+set "CXXFLAGS= -MD"
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% -DJKQtPlotter_BUILD_EXAMPLES=OFF -DJKQtPlotter_BUILD_STATIC_LIBS=OFF -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
Can you use `^` to make this multi line?

Reply:
Fixed in 72ada10a3756fa81e98a23a1e9197f5f5e6e231b.